ci-operator/config/ibmz-eco/isv-operator/ibmz-eco-isv-operator-master.yaml

```diff
@@ -0,0 +1,38 @@
+binary_build_commands: make install
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.7"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: test-e2e
+  cron: '@yearly'
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test-e2e
+      cli: latest
+      commands: |
+        unset GOFLAGS
+        make test-e2e
+      from: src
```

Comment:
@ashfreeth so here you are declaring that the `make test-e2e` command is in the `src` image which we attempt to build for you by cloning the repo which does not exist. Where are the actual tests you want to run? Are there any?

Reply:
@stevekuznetsov We just want to run a dummy test, so we can run our workflow. What image would be best to use just to do something basic like an `echo`?